submitter/biosamples.py

```diff
@@ -0,0 +1,38 @@
+from converter.biosamples import BioSamplesConverter
+from submission_broker.submission.submission import Submission, Entity
+from submission_broker.services.biosamples import BioSamples, AapClient
+
+
+class BioSamplesSubmitter:
```

Comment:
What about creating Submitter and Converter base classes? Then maybe having some kind of IoC(ish) for passing them into DirectArchiver. I'm thinking maybe pass a list of `Submitter`s and `Converter`s to DirectArchiver

Reply:
or the same as above but just a `List[BaseSubmitter]`

Reply:
Yes! I like the tuple. It looks pretty and makes it clear that they must be used together